qt_gui_cpp/CMakeLists.txt

```diff
@@ -87,6 +90,13 @@ install(TARGETS ${PROJECT_NAME}
 # be updated for ROS2 style
 
 if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+
+  # Disabling copyright test. The copyright used in this package does not conform to
+  # ament's copyright tests
```

Comment:
Is it just the formatting of the copyright headers in each file?  If so, we can update that, similar to how we recently did that in message_filters.

Reply:
https://github.com/ros-visualization/qt_gui_core/pull/295/commits/682f4e289e8a664616d2fc7df842d29aa6b5ccc9